contracts/token/ERC20/CappedToken.sol

```diff
@@ -11,7 +11,10 @@ contract CappedToken is MintableToken {
 
   uint256 public cap;
 
-  constructor(uint256 _cap) public {
+  constructor(uint256 _cap, address[] _minters)
+    public
+    MintableToken(_minters)
```

Comment:
nit: does solium complain that the `public` modifier has to be above `MintableToken` here? Convention is the inherited contract instantiation and then the visibility modifier

Reply:
It doesn't complain either if I place it first or last. We don't seem to follow that convention though, in some of our contracts visibility is indeed first, we should get a Solium rule and fix that.

Reply:
interesting. I'd support inheritances before visibility for constructors, since that seems to be the convention in contracts I've seen.